### DISCUSSION TITLE
Be welcoming for new users

### DISCUSSION THREAD

**Discussion by Lakoja** [tuskyapp/Tusky#3144] [Ideas]
I wonder if it might be worthwhile to be more helpful to new users (as Tusky).

I guess the reaction of 99% of new users to Mastodon is: Where is everybody and what can and should I do?

![Hieristnichts](https://user-images.githubusercontent.com/1618905/211012802-d5db1272-6848-438f-b2f1-ad3ed471283e.png)

3 of the 4 tabs of the initial view are empty and the local timeline has no hint what "local" is. And for smaller instances it would be disappointingly empty.

Ideas that come to mind (sorted by perceived usefulness):

* Welcome screen (with short intro text; direct link to local and federated timeline and profile; and below a glossar of what some terms i.e. federation mean. The screen is only shown for users without content/followers and can be skipped)
* Short cancellable help texts in the different tabs
* Offer help texts for some selected features (e. g. alt texts, status visibility, ...)

I guess it could be argued what actually "helpfulness" is and to what goal.
However the current approach seems to exclude nearly everybody but the most tech-savvy or determined people.

**Comment by nikclayton** [comment]
Yes. See also https://github.com/tuskyapp/Tusky/issues/3213
